src/pages/card/view/model.ts

```diff
@@ -23,3 +34,19 @@ export const $pageTitle = combine(
     return card.title;
   },
 );
+
+// Обработка события удаления карточки
+guard({
+  clock: deleteCard,
+  source: cardModel.$currentCardId,
+  filter: (id): id is string => id !== null,
+  target: deleteCardByIdFx,
+});
+
+// Возвращаем на домашнюю страницу после события удаления карточки
+sample({
+  clock: deleteCardByIdFx.done,
+  // FIXME: push user to profile
```

Comment:
[`nitpick:`](https://conventionalcomments.org/) Мб так?

```suggestion
  // FIXME: push later to card.author page
```

Reply:
согласен